draft-ietf-quic-v2.md

```diff
@@ -238,6 +259,15 @@ Change Controller: IETF
 
 Contact: QUIC WG
 
+This document also requests that IANA add the following entry to the TLS
+Application-Layer Protocol Negotiation (ALPN) Protocol IDs registry:
+
+Protocol: HTTP/3 over QUICv2
+
+Identification Sequence: 0x68 0x33 0x76 0x32 ("h3v2")
```

Comment:
We need to do the same thing that we did during h3 development, which is to add draft versions: `h3v2-03` refers to "HTTP/3 over draft-ietf-quic-v2-03"

Reply:
The next version of the draft is intended to use the final version and ALPN.

Reply:
Uh... I'd be more comfortable trying out a draft in production before we burn the codepoints. The QUIC version can always be regenerated randomly, but the ALPN codepoint would be trickier

Reply:
OK, h3q2-01